### PR TITLE
#3087 Added preassembled Ramp model of the Modelica standard library

### PR DESCRIPTION
**Pull request by joyelfeghali**
**Checklist before requesting a review**

*use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes*

- [ ] unit tests and non-regression tests were added (new model, new feature and bug fix)
- [ ] main documentation was updated (update of input/output file, 3rd party, model, repository organization, solver)
- [ ] example documentations were updated (new example in examples folder)
- [ ] the corresponding milestone was added in the ticket and in this PR
- [ ] if this PR modifies the parameters or inputs/outputs of a model/solver: the corresponding xsl was added in util/xsl and platform DB were updated
- [ ] if this PR modifies a dictionary: the corresponding french dictionary was updated

### CODE REVIEW

**Comment by joyelfeghali** [comment]
@adriguir FYI

**Review by rosiereflo — CHANGES_REQUESTED** [review]

File: nrt/data/IEEE14/IEEE14_BasicTestCases/IEEE14_LoadVariationRamp/IEEE14.jobs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    This Source Code Form is subject to the terms of the Mozilla Public
+    License, v. 2.0. If a copy of the MPL was not distributed with this
+    file, you can obtain one at http://mozilla.org/MPL/2.0/.
+    SPDX-License-Identifier: MPL-2.0
+
+    This file is part of Dynawo, an hybrid C++/Modelica open source time domain
+    simulation tool for power systems.
+--><dyn:jobs xmlns:dyn="http://www.rte-france.com/dynawo">
+  <dyn:job name="IEEE14 - Load Variation">
+    <dyn:solver lib="dynawo_SolverIDA" parFile="solvers.par" parId="2"/>
```
Comment (marcochiaramello) [inline nrt/data/IEEE14/IEEE14_BasicTestCases/IEEE14_LoadVariationRamp/IEEE14.jobs:15]: do we need to use IDA for this test?

File: nrt/data/IEEE14/IEEE14_BasicTestCases/IEEE14_LoadVariationRamp/solvers.par [hunk 2]
```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Copyright (c) 2015-2019, RTE (http://www.rte-france.com)
```
Comment (marcochiaramello) [inline nrt/data/IEEE14/IEEE14_BasicTestCases/IEEE14_LoadVariationRamp/solvers.par:3]: wrong copyright

File: nrt/data/IEEE14/IEEE14_BasicTestCases/IEEE14_LoadVariationRamp/solvers.par [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    All rights reserved.
+    This Source Code Form is subject to the terms of the Mozilla Public
+    License, v. 2.0. If a copy of the MPL was not distributed with this
+    file, you can obtain one at http://mozilla.org/MPL/2.0/.
+    SPDX-License-Identifier: MPL-2.0
+
+    This file is part of Dynawo, an hybrid C++/Modelica open source time domain
+    simulation tool for power systems.
+-->
+<parametersSet xmlns="http://www.rte-france.com/dynawo">
```
Comment (marcochiaramello) [inline nrt/data/IEEE14/IEEE14_BasicTestCases/IEEE14_LoadVariationRamp/solvers.par:14]: please indicate the parameters only for the solver you use for the test

File: nrt/data/IEEE14/IEEE14_BasicTestCases/IEEE14_LoadVariationRamp/IEEE14.dyd [hunk 4]
```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Copyright (c) 2015-2019, RTE (http://www.rte-france.com)
```
Comment (marcochiaramello) [inline nrt/data/IEEE14/IEEE14_BasicTestCases/IEEE14_LoadVariationRamp/IEEE14.dyd:3]: wrong copyright, everywhere

**Review by marcochiaramello — CHANGES_REQUESTED** [review]

File: nrt/data/IEEE14/IEEE14_BasicTestCases/IEEE14_LoadVariationRamp/IEEE14.jobs (hunk 1 above)
Comment (joyelfeghali) [reply nrt/data/IEEE14/IEEE14_BasicTestCases/IEEE14_LoadVariationRamp/IEEE14.jobs:15]: There is no difference in the ramp value when using the simplified solver or IDA solver. I just kept the solver used for the IEEE14_LoadVariation test case.

**Review by rosiereflo — APPROVED** [review]

**Pull request merged**